docs/migrating-to-velero.md

```diff
@@ -0,0 +1,50 @@
+# Migrating from Ark to Velero
+
+Users of Ark will need to [upgrade to v0.10.x][1], then move their installation to the renamed Velero images and Custom Resource Definitions (CRDs), as well as switch to the `velero` client. This is a one-time operation.
```

Comment:
Let's add a section header called `Prerequisites` and put a bullet about being on `v0.10.x` there.

Reply:
Are there any other prerequisites? I think it's only v0.10.x

Reply:
probably worth putting `kubectl`, and ~`cluster-admin` access to the cluster. We might also want to put `crd-migrator` and the `velero` release tarball in here - not sure yet if those make more sense as steps in the script or prereqs.